matrices/config.hpp.in

```diff
@@ -40,10 +40,11 @@ namespace matrices {
 
 const char *location_ani1_mtx = "@Ginkgo_BINARY_DIR@/matrices/test/ani1.mtx";
 const char *location_ani4_mtx = "@Ginkgo_BINARY_DIR@/matrices/test/ani4.mtx";
+const char *location_isai_mtx = "@Ginkgo_BINARY_DIR@/matrices/test/";
```

Comment:
nit: Since it is a directory now, maybe a namechange might be in order:
```suggestion
const char *location_isai_mtxs = "@Ginkgo_BINARY_DIR@/matrices/test/";
```